near-contract-standards/src/contract_metadata.rs

```diff
@@ -1,17 +1,171 @@
+pub use build_info::BuildInfo;
 use near_sdk::serde::{Deserialize, Serialize};
 
-/// The contract source metadata is a standard interface that allows auditing and viewing source code for a deployed smart contract.
-#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+/// The struct provides information about deployed contract's source code and supported standards.
+///
+/// Contract source metadata follows [**NEP-330 standard**](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) for smart contracts
+///
+/// See documentation of [`near_api::types::contract::ContractSourceMetadata`](https://docs.rs/near-api/latest/near_api/types/contract/struct.ContractSourceMetadata.html)
```

Comment:
There is a chance that it will be moved, and it will ruin the doc. :(
But probably, nothing we can do except to remember for that

Reply:
Maybe a link to an example but as a permanent link (to a hash), so it stay at least partially correct

Reply:
E.g.:
https://github.com/near/near-api-rs/blob/797f9fcb5e8fae3be8883d382968331888809f62/examples/contract_source_metadata.rs

Reply:
@akorchyn that's a reason not to move or commit reminder to `near-api-rs` to also update backlink update at `near-sdk-rs` wherever public import path of `ContractSourceMetadata` changes.  . I don't want to ruin the illusion of seamlessly jumping the cozy docs.rs website.